weareinreach/control-panel
# Feature - Control Panel - Export to CSV Button

## Description
Added a button that generates a CSV file of current search results when clicked.

## Asana ticket:
https://app.asana.com/0/1132189118126148/1202105700437604

## PR Checklist
- [x ] Assign @trigal2012 **and** @Alfredo-Moreira as reviewers.
- [x ] If your PR is not a hotfix, is it targeted for `dev`? If your PR is a hotfix, is it targeted for `master`?
- [x ] Complex code is well documented with comments.
- [x ] Does the original ticket have test instructions? If not add them below

## How to Test
In the control panel on the organizations page, select some search filters and perform a search. Click on the export to CSV button. All of the search filters that were applied should show up on the CSV file, along with the name of the organizations for the results.
